src/extension.ts

```diff
@@ -318,7 +319,10 @@ export async function activateInternal(
         registerCommand(showOutputChannelCommandId, () => { ext.outputChannel.show(); });
     });
 
-    return createApiProvider([]);
+    return createApiProvider([<AzureExtensionApi>{
+        apiVersion: '1.0.0',
+        createWebApp: api.createWebApp
```

Comment:
Anything in an API should have a test to accompany it to prevent us from accidentally breaking backwards compatibility